maat-court-data-api/src/main/java/gov/uk/courtdata/authorization/controller/AuthorizationController.java

```diff
@@ -0,0 +1,46 @@
+package gov.uk.courtdata.authorization.controller;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import gov.uk.courtdata.authorization.service.AuthorizationService;
+import gov.uk.courtdata.dto.ErrorDTO;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/authorization")
+@Slf4j
+@RequiredArgsConstructor
+@Tag(name = "Authorization", description = "Rest API for performing role authorization checks")
```

Comment:
Can we also create the schema definition for this API?  I think there are some Swagger/Open API docs. Can we update those, please?